### DISCUSSION TITLE
这是只能电脑下载吗？

### DISCUSSION THREAD

**Discussion by lllLinux** [xifangczy/cat-catch#180] [General]
为什么三个浏览器我都试过了，都不能下载猫抓啊![Screenshot_2023-05-27-16-55-05-14.jpg](https://github.com/xifangczy/cat-catch/assets/133392023/014a63e2-50e0-496f-ac9e-baa31144da5a)
这个是微软的，之前firefox的我也试过，也没用，有没有人给个蓝奏云或者网盘的链接？

**Comment by xifangczy** [comment]
用什么浏览器 据我所知  Kiwi 可以安装扩展。。
firefox手机版可以通过一些方法安装。

**Comment by YFHDTD** [comment]
yes
